config/constants.go

```diff
@@ -13,6 +13,8 @@ const (
 	DataDirectory = "data"
 	// EmojiDir is relative to the webroot.
 	EmojiDir = "/img/emoji"
+	// WebhookWorkerPoolSize defines the number of concurrent HTTP webhook requests.
+	WebhookWorkerPoolSize = 10
```

Comment:
Since this value is an internal detail to the worker pool itself and not to the application as a whole, maybe just move it to that file.